HW_1/hw_1_2.py

```diff
@@ -0,0 +1,24 @@
+# Напишите код, который запрашивает число и сообщает, является ли оно простым или составным.
+# Используйте правило для проверки: «Число является простым, если делится нацело только на единицу
+# и на себя». Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+# Для поиска всех простых чисел в заданном диапазоне рекомендуется применять решето Эратосфена
+# Но в данной задаче надо проверить на простоту только одно число, поэтому алгоритм будет простой с незначительной оптимизацией
+MIN_VALUE = 0
+MAX_VALUE = 100000
+while True:
+    number = int(input(f'Введите число от {MIN_VALUE} до {MAX_VALUE}: '))
+    if number >= MIN_VALUE and number <= MAX_VALUE:
+        break
+    else:
+        print('Не корректный ввод. Попробуйте ещё раз')
```

Comment:
В конструкцию `else` можно опустить, так как если `break` сработает, то 14 строка – нет
```suggestion

    print('Не корректный ввод. Попробуйте ещё раз')
```